support-frontend/assets/components/orderSummary/contributionsOrderSummaryContainer.tsx

```diff
@@ -136,8 +136,8 @@ export function ContributionsOrderSummaryContainer({
 
 	return renderOrderSummary({
 		description,
-		total: currentPrice,
-		totalOriginal: isSupporterPlus ? originalPrice : undefined,
+		total: productPriceWithPromo,
+		totalOriginal: isSupporterPlus ? productPrice : undefined,
```

Comment:
To follow the naming above, maybe `total` and `totalWithPromo`?

 ```suggestion
		total: isSupporterPlus ? productPrice : undefined,
		totalOriginal: productPriceWithPromo,
```

Reply:
Not 100% sure, but have  renamed `totalOriginal` to `totalExcludingPromo`. `total` includes the promoCode adjustment.